recipes/strct/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "strct" %}
+{% set version = "0.0.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6ff67907c530f249e5c5aa77e3d6fa0e74e3d0de1c8be8ddd1a10980ca6ed621
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
```

We're starting to encourage always having Python version constraints on `noarch: python` recipes:
```suggestion
    - python >=3.5
```